app/src/main/java/org/oppia/app/player/state/DragDropSortInteractionView.kt

```diff
@@ -26,13 +29,16 @@ import javax.inject.Inject
 class DragDropSortInteractionView @JvmOverloads constructor(
   context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0
 ) : RecyclerView(context, attrs, defStyleAttr) {
+  // Default to grouping is disabled.
```

Comment:
Can we reword this a bit? I believe this means that grouping is off by default right? Also can we provide this constant via dagger (You can see an example [here](https://github.com/oppia/oppia-android/blob/develop/domain/src/main/java/org/oppia/domain/question/QuestionTrainingConstantsProvider.kt)

Reply:
Why we would require a comment to be provided with dagger ?

Reply:
i mean this is not a constant  this is a flag which is turned on from customization arg of a state